packages/components/List/index.js

```diff
@@ -51,7 +53,17 @@ const List = ({ projectItems, handleEdit, handleDelete }) => {
   };
 
   const listItems = items.map((item) => {
-    if (item.display) {
+    console.log('item map', item);
+    if (item.display && item.status) {
```

Comment:
Will there be a situation when `item.status` is `undefined` when `display` exists?

Reply:
Yes - so basically, I refactored the List component so that it can display either the TranscriptCard component or SimpleCard component. In the original repo, there is a List component and a [ListPageTranscript component](https://github.com/bbc/digital-paper-edit-client/blob/master/src/Components/lib/ListPageTranscript/index.js), which seemed excessive to me.

I left a question about whether we might change the data structure a bit to make this less hack-y, but for right now:

- Either a TranscriptCard or a SimpleCard can have `display` set to true or false — this is toggled by the SearchBar component. 
- An item that has a `display` and a `status` is a TranscriptCard (the status pertains to the transcription job). This tells List to render a TranscriptCard rather than a SimpleCard.
- An item that has a `display` but not a `status` is a Project or Paper Edit. These need to be rendered as SimpleCards.

Again, this is real hack-y. I wasn't sure whether I could add a type field to the data (see above) to make this better.

Reply:
Awesome to know. I like this behaviour! :)

Reply:
I agree it's a thing that can be improved, but doesn't need to be done now.